packages/jsii-pacmak/test/expected.jsii-calc-lib/python/src/scope/jsii_calc_lib/__init__.py

```diff
@@ -1,3 +1,6 @@
+"""
```

Comment:
does it make sense to include an empty docstring or perhaps omit it if it's empty?